### ISSUE TITLE
Failing to import downloads looking in a subfolder that doesn't exist

### ISSUE THREAD

**Issue by cckndson** [mrusse/soularr#48]
Hello,

So I have the docker container configured and it's working 95% of the way through. It's searching and downloading, I see those downloads in slskd and in the downloads folder. However when the script looks for the download it looks for them in downloads/artist_name instead of just downloads where the files are. It then moves them to failed imports because it fails to import. I can't see anywhere in lidarr, slskd, or soularr where it should be looking at /downloads/artist_name instead of just downloads.

> [INFO|soularr|L333] 2024-12-02T18:07:22+0000: Username: xxxxxxxx Directory: Tagged\B\Britney Spears\(You Drive Me) Crazy (The Stop Remix!) [1999]
> 
> [INFO|soularr|L335] 2024-12-02T18:07:22+0000: -------------------
> 
> [INFO|soularr|L336] 2024-12-02T18:07:22+0000: Waiting for downloads... monitor at: http://x.x.x.x:5030/downloads
> 
> [INFO|soularr|L374] 2024-12-02T19:08:08+0000: Stall timeout reached! Removing stuck downloads...
> 
> [INFO|soularr|L385] 2024-12-02T19:08:08+0000: All tracks finished downloading!
> 
> [INFO|soularr|L436] 2024-12-02T19:08:14+0000: Starting Lidarr import for: Britney Spears ID: 74060
> 
> [INFO|soularr|L436] 2024-12-02T19:08:14+0000: Starting Lidarr import for: New Order ID: 74061
> 
> [INFO|soularr|L436] 2024-12-02T19:08:14+0000: Starting Lidarr import for: The Clash ID: 74062
> 
> [INFO|soularr|L451] 2024-12-02T19:08:14+0000: Downloaded Albums Scan Failed to import from: /downloads/Britney Spears
> 
> [INFO|soularr|L477] 2024-12-02T19:08:14+0000: Failed import moved to: failed_imports/Britney Spears
> 
> [INFO|soularr|L451] 2024-12-02T19:08:14+0000: Downloaded Albums Scan Failed to import from: /downloads/New Order
> 
> [INFO|soularr|L477] 2024-12-02T19:08:14+0000: Failed import moved to: failed_imports/New Order
> 
> [INFO|soularr|L451] 2024-12-02T19:08:14+0000: Downloaded Albums Scan Failed to import from: /downloads/The Clash
> 
> [INFO|soularr|L477] 2024-12-02T19:08:14+0000: Failed import moved to: failed_imports/The Clash

**Comment by mrusse** [comment]
Well the script moves and renames the folders so everything is in an artist folder. Lidarr will only import the tracks if they are in folders that match the artist name.

Is that not happening for you?

**Comment by IncompatibleW** [comment]
I'm having the same issue. Downloaded media is not being sorted into artist folders, and instead is just failing import each time. I'm not sure if this is a Soularr issue or a Lidarr one, but I am getting the same "Failed to import" logs as the OP is.

**Comment by cckndson** [comment]
I'll get the info to you when I'm back in front of my PC. I can confirm they aren't being moved into the lidarr import folder. It'll be probably an hour before I'm back in front of it and can send all the info.

**Comment by cckndson** [comment]
I figured it out on my end. slskd saves to a subfolder in my downloads and I had to put that specific subfolder in the lidarr download_dir field in the config.ini instead of just the overall downloads folder that all my downloaders are mapped to. Once I updated that it began to import correctly. 

So for example instead of /downloads it needed to be /downloads/slskd under lidarr. Nothing had to be changed under slskd.
